bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -4,6 +4,8 @@ apiVersion: apps/v1
 metadata:
   {{ if eq .OVN_NODE_MODE "dpu-host" }}
   name: ovnkube-node-dpu-host
+  {{ else if eq .OVN_NODE_MODE "smart-nic" }}
```

Comment:
dpu-host mode takes precedence than smart-nic mode if both labels are configured, should we add a warning in ovn bootstrap that indicates that dpu-host mode is chosen?

Reply:
Very good catch! I didn't think of this. I added code such that we error if any nodes have overlapping labels.

Reply:
I think I didn't completely solve this. Working on it.

Reply:
@zshi-redhat I had to also enable the watch on node updates to get this working. I couldn't find another way.